local-volume/provisioner/pkg/util/api_util.go

```diff
@@ -98,8 +99,9 @@ func (u *FakeAPIUtil) DeletePV(pvName string) error {
 		u.deletedPVs[pvName] = pv
 		delete(u.createdPVs, pvName)
 		u.cache.DeletePV(pvName)
+		return nil
 	}
-	return nil
```

Comment:
Can we please create an errList with multiple NewNotFound errors, and return utilerrors.NewAggregate(errList)? This change will mirror the return in volume_util.go:L123.

Additionally, it might be worth creating a test case where only one of the errors in the list is a NewNotFound error.

Reply:
@ianchakeres I donot think a errList is needed, since here we only result at most one error.